qiskit/ignis/experiments/calibration/instruction_data/gate_and_schedules_definition.py

```diff
@@ -19,31 +34,39 @@ class InstructionsDefinition:
     The class allows users to define single-pulse schedules as quantum circuits and
     then leverage the quantum circuit compose operation to compose more complex
     circuits from basic circuits.
-
-    # TODO
-    - test recursive behavior of instructions
     """
-
-    def __init__(self, backend_name: str, n_qubits: int, channel_qubit_map: Dict,
-                 pulse_table: Optional[PulseTable] = None):
+    TABLE_COLS = ['gate_name', 'qubits', 'signature', 'gate_id', 'program']
```

Comment:
Is signature intended to allow scoping? Perhaps a better name would be needed? `scope`?

Reply:
`signature` is to define a list of parameters for `InstructionScheduleMap`, ie `U2(P0, P1)`. I avoided to use `parameters` because what we are going to calibrate is also parameter.

Reply:
Oh I see. Can we somehow change the name to better express this? I agree that `parameters` is misleading. How about `InstMapSignature`, `InstMapName`, or `InstMapCode`? Do you have better ideas?